test/test_transforms.py

```diff
@@ -462,6 +462,16 @@ def test_resize_size_equals_small_edge_size(height, width):
     result = t(img)
     assert max(result.size) == max_size
 
+@pytest.mark.parametrize("height, width", [(28, 28)])
+def test_resize_equal_input_output_sizes(height, width):
+    # Regression test for https://github.com/pytorch/vision/issues/7518
```

Comment:
Why parametrize if we only have one set of parameters? Plus, let's use different height and width to harden against any future regressions.

```suggestion
def test_resize_equal_input_output_sizes(height, width):
    # Regression test for https://github.com/pytorch/vision/issues/7518
    height, width = 28, 27
```